versionpb/version.pb.go

```diff
@@ -838,6 +839,13 @@ func (x *Version) GetImageHash() string {
 	return ""
 }
 
+func (x *Version) GetImageHashArm64() string {
```

Comment:
I'm not sure should we have 2 separate functions or one function which will ask for `arch` parameter.

Reply:
this is a generated file, I'm not sure if we should modify it

Reply:
yep, my bad it is generated